### ISSUE TITLE
Import path is incorrect.

### ISSUE THREAD

**Issue by Tanimodori** [Tanimodori/viteburner#1]
```ts
import { NS } from '@ns';
import { relative } from './relative';
import { absolute } from '@/import/absolute';

export async function main(ns: NS) {
  relative(ns);
  absolute(ns);
}
```

```js
import { relative } from "/src/import/relative.ts";
import { absolute } from "/src/import/absolute.ts";
export async function main(ns) {
  relative(ns);
  absolute(ns);
}
```

Vite can serve `/src/import/relative.ts` but bitburner cannot. Need a transform here.

**Comment by Tanimodori** [comment]
Reference: https://github.com/vitejs/vite/blob/main/packages/vite/src/node/plugins/importAnalysis.ts

**Comment by Tanimodori** [comment]
Workaround (for JS only):

1. In `vite.config.ts`, set the `transform` of js files to `false`, so it won't be touched by vite.
2. Use relative imports only. Avoid using absolute imports like `@/foo.js` as it will not be transformed by `vite` now and obviously not working with `bitburner`.

**Comment by Tanimodori** [comment]
TODO: transform import path for each server and combined sourcemap

**Comment by Tanimodori** [comment]
fixed in 07d083d5f6c729e08f02beff3413a514dcf2ced3
